validations/users.js

```diff
@@ -0,0 +1,82 @@
+const { getUsers } = require('../utils/storage');
+
+function validateUserName(userName) {
+    if (!userName) {
+        throw new Error('Error. There was no username provided.');
+    }
+}
+
+function validatePassword(password) {
+    if (!password) {
+        throw new Error('Error. There was no password provided.');
+    }
+}
+
+function findUserWithSameUserNameAndPassword(userName, password) {
+    let users = getUsers();
+
+    const user = users.find(
+        (user) => user.userName === userName && user.password === password
+    );
+
+    if (!user) {
+        throw new Error('Error. No such user.');
+    }
+
+    return user;
+}
+
+function validateLoginSession(userLoginSession) {
+    if (!userLoginSession) {
+        throw new Error('Error. There was no users session data provided.');
+    }
+}
+
+function findUserByUserLoginSession(userLoginSession) {
+    let users = getUsers();
+
+    const user = userLoginSession
+        ? users.find((user) => user.session === userLoginSession)
+        : null;
+
+    if (!user) {
+        throw new Error('You are not logged in.');
+    }
+
+    return user;
+}
+
+function getUser(userId) {
```

Comment:
The `getUser` function should not be in `validations` module

Reply:
Replaced.